app/brave_settings_strings.grdp

```diff
@@ -1378,6 +1378,15 @@ Leo. This action can't be undone.
   <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_MODEL_API_KEY_TOOLTIP_INFO" desc="A brief description for the input title for adding API key">
     Some serving frameworks may require authentication credentials, such as an API key or an access token. These will be added to the request header.
   </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_CONTEXT_SIZE" desc="An input title for context size, which refers to the maximum number of tokens (small units of text, usually a few characters or words) that an AI model can process in one go. The context size determines how much data the model can consider simultaneously. For example, the Phi 3.5 model can process up to 131072 tokens in its context window.">
+    Context size
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_CONTEXT_SIZE_TOOLTIP_INFO" desc="A tooltip explaining the concept of context size. Context size is the number of tokens, each representing a small part of the text, such as a few characters or words. A larger context size allows the model to retain more information and maintain coherence over longer conversations. (4 tokens ≈ 3 words)">
+    The number of tokens representing the model's context window, affecting its ability to maintain coherence in longer interactions (4 tokens ≈ 3 words).
+  </message>
+  <message name="IDS_SETTINGS_LEO_ASSISTANT_INPUT_CONTEXT_SIZE_NOTE" desc="A note for users explaining how to find the appropriate context size for their AI model. Context size refers to the number of tokens an AI model can handle. Users can check their model documentation to find this value. Ollama users can retrieve this information using the `ollama show ...` command. If no value is provided, Brave will assume a default context size.">
+    Consult your model for the appropriate value. Ollama users can use the `ollama show …` command to retrieve this information. By default, Brave will assume a context size of <ph name="DEFAULT_CONTEXT_SIZE">$1<ex>4k</ex></ph> tokens.
```

Comment:
The example assumes we want people to type in the format "4k" but I don't think we'll be doing that. I know we're replacing that with the actual value so not sure it matters, but it might slightly matter to translators?

Reply:
Users will only be able to enter digits, but this is a good point. I'll switch to 4000.